source/taup/taup_velplot.rst

```diff
@@ -0,0 +1,27 @@
+taup velplot
+============
+
+:本节贡献者: |田冬冬|\（作者）、
+             |姚家园|\（审稿）
+:最近更新日期: 2021-06-01
+
+----
+
+``taup velplot`` 可以很方便地生成用于绘制速度-深度剖面的 GMT 脚本。
+
+使用 ``taup velplot --help`` 可查询该命令的完整用法和选项。此处仅介绍其常用用法。
+
+绘制 iasp91 参考模型的速度-深度剖面::
```

Comment:
The command is not uses to plot vel-depth but output a script to plot it.

Reply:
Please see my suggestion below,.